07_purity_prediction/download_purity.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>Is there a reason you chose a stripplot here over a violin plot? Is the purpose to show the differences in ranges of the different cancer types?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I think either would be fine. I just generally wanted to see if any of the purity distributions are particularly skewed or if any cancer types are drastically under-represented compared to the others (the latter seems a bit easier to see with a strip plot, although I guess I could make a violin plot with the sample sizes for each cancer type as labels). This was just an exploratory visualization, though, so I don't want to spend too much time tweaking it right now.</p>